src/components/album/CoverImage.css

```diff
@@ -0,0 +1,65 @@
+.coverImg{
+  width: 100%;
+  /* border: 1px red solid;     */
+}
+
+.imgContainer{
+  position: relative;
+  display: inline-block;
+}
+
+.imgFunction{
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  opacity: 0;
+  background: rgba(0, 0, 0, 0.4);
+  transition: opacity 0.3s ease;
```

Comment:
Nice transition! Would be nice with a transition on the growing of the play icon too 🤩